components/brave_perf_predictor/browser/bandwidth_linreg_parameters.h

```diff
@@ -0,0 +1,812 @@
+  "LoopMe",
+  "Quantcast",
+  "Click4Assistance",
+  "Hotjar",
+  "Snapchat",
+  "jQuery CDN",
+  "Segment",
+  "Usabilla",
+  "Nativo",
+  "Sharethrough",
+  "Twitter Online Conversion Tracking",
+  "BounceX",
+  "Integral Ad Science",
+  "Rubicon Project",
+  "Index Exchange",
+  "Sentry",
+  "Cloudflare CDN",
+  "VigLink",
+  "Optimizely",
+  "Ensighten",
+  "Criteo",
+  "Nielsen NetRatings SiteCensus",
+  "Cookie-Script.com",
+  "Rackspace",
+  "Adobe TypeKit",
+  "Stripe",
+  "Trust Pilot",
+  "Polyfill service",
+  "Affiliate Window",
+  "FontAwesome CDN",
+  "Bootstrap CDN",
+  "Auto Link Maker",
+  "Embedly",
+  "JSDelivr CDN",
+  "OneSignal",
+  "The Trade Desk",
+  "Instagram",
+  "PayPal",
+  "Taboola",
+  "Opentag",
+  "Brightcove",
+  "VWO",
+  "Rambler",
+  "Media Math",
+  "Google Maps",
+  "Unpkg",
+  "Yandex Share",
+  "Yandex Metrica",
+  "Yandex CDN",
+  "Amplitude Mobile Analytics",
+  "Yahoo!",
+  "Yandex Ads",
+  "piano",
+  "Moat",
+  "Parse.ly",
+  "Unruly Media",
+  "Skimbit",
+  "ZenDesk",
+  "Silverpop",
+  "AddThis",
+  "Polldaddy",
+  "Dailymotion",
+  "Disqus",
+  "Alexa",
+  "Mailchimp",
+  "Tealium",
+  "LiveChat",
+  "DemandBase",
+  "Tencent",
+  "Oracle Recommendations On Demand",
+  "Mixpanel",
+  "PerimeterX Bot Defender",
+  "Evidon",
+  "Media.net",
+  "Ghostery Enterprise",
+  "LongTail Ad Solutions",
+  "Sailthru",
+  "Marketplace Web Service",
+  "Pinterest",
+  "BrightTag / Signal",
+  "mPulse",
+  "ForeSee",
+  "Permutive",
+  "FirstImpression",
+  "Connatix",
+  "Media Management Technologies",
+  "Mobify",
+  "Yieldify",
+  "Crazy Egg",
+  "SurveyMonkey",
+  "Touch Commerce",
+  "RichRelevance",
+  "Reevoo",
+  "Micropat",
+  "Playbuzz",
+  "Po.st",
+  "Fastly",
+  "eBay",
+  "TRUSTe",
+  "Qualtrics",
+  "Aggregate Knowledge",
+  "Digioh",
+  "Gigya",
+  "Crowd Control",
+  "LinkedIn Ads",
+  "Riskified",
+  "BlueKai",
+  "AMP",
+  "eXelate",
+  "Captify Media",
+  "Hola Networks",
+  "Polar Mobile Group",
+  "Apester",
+  "StreamRail",
+  "SpringServer",
+  "Monetate",
+  "Adobe Scene7",
+  "Opta",
+  "FLXone",
+  "Sift Science",
+  "Accuweather",
+  "Lucky Orange",
+  "AWeber",
+  "Salesforce.com",
+  "Wistia",
+  "Histats",
+  "ShareThis",
+  "Adyoulike",
+  "Pusher",
+  "PERFORM",
+  "Pingdom RUM",
+  "Cloudflare",
+  "Hubspot",
+  "Curalate",
+  "Decibel Insight",
+  "Mouseflow",
+  "Symantec",
+  "Proper Media",
+  "Vimeo",
+  "LivePerson",
+  "Clicktale",
+  "iPerceptions",
+  "Sekindo",
+  "OpenX",
+  "Teads",
+  "sovrn",
+  "GumGum",
+  "Microsoft Hosted Libs",
+  "Vox Media",
+  "Concert",
+  "Xaxis",
+  "unpkg",
+  "Maxymiser",
+  "Kaltura Video Platform",
+  "SoundCloud",
+  "Kargo",
+  "TripleLift",
+  "Trip Advisor",
+  "Cloudinary",
+  "Tawk.to",
+  "Klevu Search",
+  "Yandex APIs",
+};
+
+const base::flat_set<std::string> relevant_entity_set(relevant_entities);
+
+struct stdfactor {
+    double mean, scale;
+};
+
+const std::map<std::string, stdfactor> stdfactor_map = {
```

Comment:
CC prohibits using global vars with non-trivial destructor... But probably writing all `base::NoDestructor` boilerplate would be too much here. Maybe just place a todo.
For now we can replace `const std::array` with `constexpr std::array` and also switch `vector` to `array`

Reply:
I guess we could also wrap all parameters up in a class and expose as static?

Reply:
not really :-/ The standard way is something like this https://cs.chromium.org/chromium/src/content/browser/frame_host/navigation_request.cc?q=NoDestructor%3Cstd::map&sq=package:chromium&dr=C&l=154

Reply:
`constexpr` values are OK

Reply:
added constexpr where possible, `constexpr std::array` doesn't work for `std:array<std::string, size>` though. Tried adding some of the `static const base::NoDestructor` boilerplate, but don't think it makes much of a difference if these stay at global scope anyway?

Reply:
yeah, moreover it is banned
```
// Furthermore, a NoDestructor<T> should never
// have global scope as that may require a static initializer
```